### PR TITLE
Add top stat info to mongodb plugin

### PR DESCRIPTION
**Pull request by denispershin**
### Required for all PRs:

- [x] Associated README.md updated.
- [x] Has appropriate unit tests.

### Changes:

Added information (insert, update, queries, remove etc...) about each collection in replica set.
More information in [documentation](https://docs.mongodb.com/manual/reference/command/top/)

### CODE REVIEW

**Comment by denispershin** [comment]
!signed-cla

File: plugins/inputs/mongodb/mongodb.go [hunk 1]
```diff
@@ -53,6 +54,9 @@ var sampleConfig = `
   ## When true, collect per collection stats
   # gather_col_stats = false
 
+  ## When true, collect stats per collection
```
Comment (srebhan) [inline plugins/inputs/mongodb/mongodb.go:57]: Can you please be a bit clearer on what exactly is collected? The text almost matches the one of `gather_col_stats`...

File: plugins/inputs/mongodb/mongodb_data.go [hunk 2]
```diff
@@ -264,6 +265,27 @@ var ColDataStats = map[string]string{
 	"ok":               "Ok",
 }
 
+var TopDataStats = map[string]string{
```
Comment (srebhan) [inline plugins/inputs/mongodb/mongodb_data.go:268]: Please make this a private variable as it is only used within this package. If you want to improve the code, you might also want to change the variables above...

File: plugins/inputs/mongodb/mongodb_server.go [hunk 3]
```diff
@@ -257,6 +271,15 @@ func (s *Server) gatherData(acc telegraf.Accumulator, gatherClusterStatus bool,
 		}
 	}
 
+	topStatData := &TopStats{}
+	if gatherTopStat {
+		topStats, err := s.gatherTopStatData()
+		if err != nil {
+			s.Log.Debugf("Unable to gather top stat data: %s", err.Error())
```
Comment (srebhan) [inline plugins/inputs/mongodb/mongodb_server.go:278]: In the `gatherColStats` path the plugin returns on error. Why do you continue here?

**Review by srebhan — COMMENTED** [review]
Only very minor comments in the code. Please take a look and let me know once you addressed them.

File: plugins/inputs/mongodb/mongodb_server.go (hunk 3 above)
Comment (denispershin) [reply plugins/inputs/mongodb/mongodb_server.go:278]: My fail, fixed.

File: plugins/inputs/mongodb/mongodb_data.go (hunk 2 above)
Comment (denispershin) [reply plugins/inputs/mongodb/mongodb_data.go]: done

File: plugins/inputs/mongodb/mongodb.go (hunk 1 above)
Comment (denispershin) [reply plugins/inputs/mongodb/mongodb.go]: done

**Review by srebhan — APPROVED** [review]
Looks good to me. Thank you!

**Review by srebhan — APPROVED** [review]
Thanks missed this one...

**Review by helenosheaa — APPROVED** [review]
Looks good to me

**Pull request merged**